python_modules/dagster-webserver/dagster_webserver/cli.py

```diff
@@ -232,6 +233,7 @@ def dagster_webserver(
             )
 
 
+@contextlib.asynccontextmanager  # type: ignore
```

Comment:
whats the type error?

Reply:
looks like if I manually type it as `AsyncIterator`, errors go away